src/webapp/cypress/integration/public/password-reset.js

```diff
@@ -67,7 +67,9 @@ sizes.forEach(size => {
                         cy.get('button[type=submit]').click()
                     })
 
-                    cy.get('[data-test=reset-password]').contains("Passwords don't match!")
+                    cy.get('[data-test=reset-password]')
+                        .should("exist")
+                        .contains("Passwords don't match")
```

Comment:
Cypress is sometimes weird about asserting on punctuation within strings.

Reply:
Maybe it thinks we're laughing at our own jokes and wants to punish us accordingly.